src/chains.ts

```diff
@@ -69,9 +60,14 @@ const pgn = {
 }
 
 export const allowedNetworks: { [chainId: number]: chains.Chain }= {
-  [999]: chains.zoraTestnet,
-  [999999999]: chains.zoraSepolia,
-  [424]: pgn,
+  [chains.zoraTestnet.id]: chains.zoraTestnet,
+  [chains.zoraSepolia.id]: chains.zoraSepolia,
+  [chains.arbitrumGoerli.id]: chains.arbitrumGoerli,
+  [chains.arbitrumSepolia.id]: chains.arbitrumSepolia,
+  [chains.arbitrum.id]: chains.arbitrum,
+  [chains.base.id]: chains.base,
+  [chains.baseSepolia.id]: chains.baseSepolia,
```

Comment:
could you add op chains too while were at it?

Reply:
Sure